src/app/modals/property-management-modal/property-management-modal.component.html

```diff
@@ -0,0 +1,162 @@
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+            <app-error-message class="error-message" [control]="form.controls.customer"></app-error-message>
+          </div>
+
+          <label for="seller" class="fw-bold">Seller</label>
+          <div class="col-md-4 w-100">
+            <select class="form-select" aria-label="Seller select" formControlName="seller">
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+            <app-error-message class="error-message" [control]="form.controls.seller"></app-error-message>
+          </div>
+
+          <label for="propertyType" class="fw-bold">Property Type</label>
+          <div class="col-md-4 w-100">
+            <select class="form-select" aria-label="Propertype Select" formControlName="propertyType">
+              <option value="1">Garage</option>
+              <option value="2">Room</option>
+              <option value="3">Studio</option>
+              <option value="4">Apartment</option>
+              <option value="5">House</option>
+            </select>
+            <app-error-message class="error-message" [control]="form.controls.propertyType"></app-error-message>
+          </div>
+
+          <label for="city" class="fw-bold">City</label>
+          <div class="col-md-4 w-100">
+            <select class="form-select" aria-label="City select" formControlName="city">
+              <option selected>Open this select menu</option>
+              <option value="1">One</option>
+              <option value="2">Two</option>
+              <option value="3">Three</option>
+            </select>
+            <app-error-message class="error-message" [control]="form.controls.city"></app-error-message>
+          </div>
+
+          <label for="typology" class="fw-bold">Typology</label>
+          <div class="col-md-4 w-100">
+            <select class="form-select" aria-label="Typology select" formControlName="typology">
+              <option value="1">T0</option>
+              <option value="2">T1</option>
+              <option value="3">T2</option>
+              <option value="4">T3</option>
+              <option value="5">T4</option>
+              <option value="6">T5</option>
+            </select>
+            <app-error-message class="error-message" [control]="form.controls.typology"></app-error-message>
+          </div>
+
+          <div> <!-- this div will force the checkbox to a column format // comment done by Pedro Cruz -->
+            <label for="Amenities" class="fw-bold">Amenities</label>
+            <div class="form-check col-md-4 w-100">
+              <input class="form-check-input" type="checkbox" formControlName="amenities" required>
+              <label class="form-check-label" for="flexCheckDefault">Laundry</label>
```

Comment:
replace "for="flexCheckChecked" with something meaningful please or remove it
https://www.w3schools.com/tags/att_label_for.asp